tests/container_disk_test.go

```diff
@@ -128,6 +131,66 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				}
 			})
 		})
+		Context("[Serial]should obey the disk verification limits in the KubeVirt CR", func() {
+			AfterEach(func() {
+				// Restoring the diskVerificaton memory usage limit
+				kv := util.GetCurrentKv(virtClient)
+				kv.Spec.Configuration.DeveloperConfiguration.DiskVerification = nil
+				_, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Update(kv)
+				Expect(err).To(BeNil())
+			})
+
+			It("disk verification should fail when the memory limit is too low", func() {
+				By("Reducing the diskVerificaton memory usage limit")
+				kv := util.GetCurrentKv(virtClient)
+				kv.Spec.Configuration.DeveloperConfiguration = &v1.DeveloperConfiguration{
+					DiskVerification: &v1.DiskVerification{
+						MemoryLimit: resource.NewScaledQuantity(42, resource.Kilo),
+					},
+				}
+				_, err := virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Update(kv)
```

Comment:
There is a helper which updates the config and waits for it to be recognized across the whole kubevirt installation. Without that the test will be flaky, because virt-handler may not yet have picked up the new change.

Just replace this with

```
kvc := util.GetCurrentKv(virtClient)

Reply:
added, thank you!